exla/lib/exla/defn.ex

```diff
@@ -844,6 +845,11 @@ defmodule EXLA.Defn do
   @bin_op [:add, :subtract, :multiply, :min, :max, :remainder, :pow, :divide, :atan2] ++
             [:bitwise_and, :bitwise_or, :bitwise_xor, :left_shift]
 
+  defp to_operator(op, [%EXLA.MLIR.Value{} = left, %EXLA.MLIR.Value{} = right], out, state)
+       when op in @bin_op do
+    apply(EXLA.MLIR.Value, op, [left, right])
+  end
```

Comment:
In theory, we just need to add the MLIR NIFs for all the `@bin_op`s and they'll be ready :)

Reply:
We should leave it to after this and sm-metal are merged to main